.vault-config/product-construction-dev.yaml

```diff
@@ -26,6 +26,14 @@ secrets:
         name: BotAccount-dotnet-bot
       gitHubBotAccountName: dotnet-bot
 
+  BotAccount-dotnet-bot-repo-dev-PAT:
```

Comment:
Do we need this for the frontend? For the VMR page?

Reply:
Oh, it was needed for the VMR page, we can kill this